src/sgame/sg_bot_ai.cpp

```diff
@@ -888,7 +885,7 @@ AINodeStatus_t BotActionFight( gentity_t *self, AIGenericNode_t *node )
 
 	if ( mind->botSkill.level >= 3 && goalDist < Square( MAX_HUMAN_DANCE_DIST )
 	        && ( goalDist > Square( MIN_HUMAN_DANCE_DIST ) || mind->botSkill.level < 5 )
-	        && primaryWP != WP_PAIN_SAW && primaryWP != WP_FLAMER )
+	        && primaryInRange )
```

Comment:
Is moving backward with the painsaw desired?

Reply:
Yes and no. I guess what would be desired is to go backward as long as your range is superior to your opponent's .

Reply:
I doubt that the painsaw range being lower than the opponents is even possible... but then that means the special case is not necessary, so I guess it's OK.